### PR TITLE
add dummy authId

### PR DESCRIPTION
**Pull request by TrevorOctober**

### CODE REVIEW

File: mentorship/comms/models.py [hunk 1]
```diff
@@ -116,3 +116,14 @@ def make_call(self, caller_id, receiver_id):
         }
 
         make_post_request.delay(url, data)
+
+    def send_message(self, recipient_id, content):
+        url = self.message_url
+
+        data = {
+            'ai': self.schedule_id,
```
Comment (KaitCrawford) [inline mentorship/comms/models.py]: this needs to be self.auth_id. while schedule_id and auth_id will both be provided by OnionDev we have no reason to believe that they'd be the same.
You can add auth_id as a field on the OnionDevApi model the same way I added schedule_id

File: mentorship/comms/models.py [hunk 2]
```diff
@@ -104,6 +104,11 @@ class OnionDevApi(Api):
         help_text="A unique integer for identifying a request, provided by "
         "OnionDev")
 
+    authId = models.IntegerField(
+        max_length=200,
+        help_text="A unique integer for "
+        "identifying a request, provided by OnionDev")
```
Comment (KaitCrawford) [inline mentorship/comms/models.py]: Make the help_text a little more specific. Something like "A unique integer for identifying message requests..."

**Comment by TrevorOctober** [comment]
@KaitCrawford @hodgestar Please review

File: mentorship/comms/models.py [hunk 3]
```diff
@@ -91,4 +91,44 @@ def send_message(self, sender_id, recipient_id, content):
...
+    """
+    Class to handle data being passed to and from OnionDev.
+    """
+
+    schedule_id = models.CharField(
+        max_length=200,
+        help_text="A unique integer for identifying a request, provided by "
+        "OnionDev")
+
+    authId = models.IntegerField(
```
Comment (hodgestar) [inline mentorship/comms/models.py]: I think this field should be called `auth_id` for consistency with our other field names?

**Comment by hodgestar** [comment]
@TrevorOctober Could you merge develop into this branch so that we can see just the changes you've made?

**Comment by hodgestar** [comment]
Once we're close to being done (or possibly in a separate PR) we'll need to add a migration that adds the new fields.

**Comment by KaitCrawford** [comment]
Whoops, good point @hodgestar! I'm going to have to add another field for the feedback surveys so I'll do it at the end of that PR

**Comment by TrevorOctober** [comment]
@hodgestar @KaitCrawford Please review

**Comment by KaitCrawford** [comment]
:+1:

**Pull request merged**